### PR TITLE
Chore/add requestor role ids

### PR DESCRIPTION
**Pull request by atharvar28**
Jira Ticket: [HP-835](https://ctds-planx.atlassian.net/browse/HP-835)

### Improvements

* Add Requestor test for request with resource_paths+role_ids parameters.

### CODE REVIEW

**Comment by paulineribeyre** [comment]
@atharvar28 @george42-ctds is this ready for review?

**Comment by george42-ctds** [comment]
@paulineribeyre @atharvar28 Yes, ready for review!

File: services/apis/requestor/requestorTasks.js [hunk 1]
```diff
@@ -21,24 +21,44 @@ module.exports = {
...
+  * @param {string} data.adminUserTokenHeader - headers for user authorized in Requestor
+  * @param {string} data.username - username to grant/revoke access for
+  * @param {string} [data.policyID] - policyID of the policy to request/revoke access
+  * @param {array} [data.resourcePaths=null] - resource_paths to request/revoke access
+  * @param {array} [data.roleIds=null] - role_ids to request/revoke access
+  * @param {boolean} [data.revoke=false] - set to true to create a revoke request
+  * @param {string} [data.requestStatus=null] - set to 'SIGNED' to approve request
+  */
+  async createRequest({
+    adminUserTokenHeader, username, policyID,
```
[the remark this answers is not in the document]
Comment (george42-ctds) [reply services/apis/requestor/requestorTasks.js:36]: Right, any fields not specified in the object are set to undefined when the function is called. 

We could set the policy_id default for consistency with resource_paths and role_ids. That might look nicer.

File: services/apis/requestor/requestorTasks.js [hunk 2]
```diff
@@ -21,24 +21,44 @@ module.exports = {
...
-  async createRequestForPolicyID(
-    adminUserTokenHeader, username, policyID, revoke = false, requestStatus = null,
-  ) {
-    console.log(`### creating request for a policy id: ${policyID} with revoke set as ${revoke}`);
-    const endPoint = revoke ? `${requestorProps.endpoint.requestEndPoint}?revoke` : `${requestorProps.endpoint.requestEndPoint}`;
-    const data = {
-      username,
-      policy_id: policyID,
-    };
+  * @param {Object} data - Pass request data as an object, include policyID or resourcePaths+roleIds
```
[the remark this answers is not in the document]
Comment (george42-ctds) [reply services/apis/requestor/requestorTasks.js:26]: The intent was to increase readability. 

With the positional parameters, the argument list was getting long and some args needed to be set as `undefined`, for example:
`createRequest(accessTokenHeader, users.user0.username, 'random-policy', undefined, undefined, true, 'SIGNED')`

**Review by paulineribeyre — APPROVED** [review]

**Review by haraprasadj — APPROVED** [review]
+1

**Review by george42-ctds — APPROVED** [review]
Looks good.

**Review by haraprasadj — APPROVED** [review]

**Pull request merged**